lib/freedom_patches/raw_handlebars.rb

```diff
@@ -54,22 +54,74 @@ def compile_handlebars(string)
 end
 
 class Ember::Handlebars::Template
-  include Discourse::Ember::Handlebars::Helper
+  prepend Discourse::Ember::Handlebars::Helper
 
-  def precompile_handlebars(string, input = nil)
-    "requirejs('discourse-common/lib/raw-handlebars').template(#{Barber::Precompiler.compile(string)});"
+  def path_for(module_name, config)
+    # We need this for backward-compatibility reasons.
+    # Plugins may not have an app subdirectory.
+    template_path(module_name, config).inspect.gsub('discourse/templates/', '')
   end
 
-  def compile_handlebars(string, input = nil)
-    "requirejs('discourse-common/lib/raw-handlebars').compile(#{indent(string).inspect});"
+  def global_template_target(namespace, module_name, config)
+    "#{namespace}[#{path_for(module_name, config)}]"
   end
 
-  def global_template_target(namespace, module_name, config)
-    # We need this for backward-compatibility reasons.
-    # Plugins may not have an app subdirectory.
-    path = template_path(module_name, config).inspect.gsub('discourse/templates/', '')
+  def call(input)
```

Comment:
Unfortunately we have to monkey patch `call` in this case. There's no way around this because the old way expected a `x = y` format and we need a method call.